test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XplatSignTests.cs

```diff
@@ -426,7 +427,56 @@ public void SignCommandArgParsing_ValidArgsContainsCertPathAsync_Succeeds()
                 });
         }
 
-        private void SignCommandArgs(Action<Mock<ISignCommandRunner>, CommandLineApplication, Func<LogLevel>, Func<SignArgs>> verify)
+        [Theory]
+        [InlineData("89967D1DD995010B6C66AE24FF8E66885E6E03A8")]
+        public void SignCommandArgParsing_LogsAWarningForInsecureCertificateFingerprint(string certificateFingerprint)
```

Comment:
Is this test missing more test cases?  It seems unusual to use a parameterised test, with only a single case.  Having said that, it looks like this test only checks SHA1 hashes, so there's no need for any other test cases, so a `[Fact]` is sufficient.

Reply:
Good catch. VS Copilot refactored the code for me, and I forgot to edit this.

Reply:
Fixed in the latest commit.